contents/newsletter/remote-working.md

```diff
@@ -0,0 +1,97 @@
+featuredImageType: full
+tags:
+  - Culture
+crosspost:
+  - Founders
+  - Blog
+---
+
+Remote work sucks. If you work at Dell, that is. The company [recently announced](https://www.businessinsider.com/dell-remote-workers-promotion-return-office-push-flexible-work-2024-3) that remote employees would not be considered for promotions or be able to change roles. Dell’s rationale, something to do with “in-person connections” and “value differentiation,” is as tired as an entry-level Dell Inspiron trying to run _Cyberpunk 2077_.
+
+But the policy actually makes sense for Dell. Sort of. When you’re a decades-old behemoth built around people working in offices, implementing remote work policies that actually work is hard. But it doesn't have to be. There are some key principles that any company – big or small, young or old, or Dell – can follow to realize the potential of remote working. 
+
+**This week, we explain: **how to make remote work actually work.
+
+> This article was first published in our newsletter, [Product for Engineers](https://newsletter.posthog.com/). It's all about helping engineers and founders build better products by learning product skills. [Subscribe here](https://newsletter.posthog.com/subscribe).
+
+### The problem? You’ve got a two-tiered system
+
+If you’re a company built around [big](https://www.cnbc.com/2023/06/08/google-to-crack-down-on-hybrid-work-asks-remote-workers-to-reconsider.html), [shiny](https://gizmodo.com/apple-remote-work-return-to-office-hybrid-work-1850256210) [offices](https://www.cnn.com/2022/06/01/tech/elon-musk-tesla-ends-work-from-home/index.html), then chances are you have a two-tiered system. Essentially, an employee’s ability to move up in such a company is closely linked to their proximity to the biggest, shiniest office. Dell’s new policy acknowledges this outright: if you’re a remote worker at Dell, there’s no career progression. The same is true at many companies that have a mix of remote, hybrid, and office workers.
+
+### The solution? Aggressive transparency
+
+Rather than presentism and nepotism, career progression should be linked to a transparent framework of measurable goals. Do the work, do it well, deliver value for customers, and you’ll be rewarded. Leif Singer, a software engineer at Automattic, the company behind WordPress, has shared some great examples of [how this works in practice](https://youtu.be/aRiP72kRrnc?feature=shared). 
+
+A two-tiered system is built on the bizarre notion that presenteeism is how you get ahead. Clearly defining and then publishing your system for career progression is an important first step to making sure that everyone, regardless of location, is assessed using the same objective criteria. At PostHog, the way we approach [career progression](https://posthog.com/handbook/people/career-progression) and [feedback](https://posthog.com/handbook/people/feedback) is laid out in public for anyone to see.
```

Comment:
Once this is moved to Substack, you can remove `https://posthog.com` from all the links so that they are relative. 
```suggestion
A two-tiered system is built on the bizarre notion that presenteeism is how you get ahead. Clearly defining and then publishing your system for career progression is an important first step to making sure that everyone, regardless of location, is assessed using the same objective criteria. At PostHog, the way we approach [career progression](https://posthog.com/handbook/people/career-progression) and [feedback](https://posthog.com/handbook/people/feedback) is laid out in public for anyone to see.
```